soccer_vision_msgs/msg/Goalpost.msg

```diff
@@ -0,0 +1,42 @@
+# Describes a detected goalpost from vision modules, in image coordinates.
+
+uint8 SIDE_UNKNOWN=0
+uint8 SIDE_LEFT=1
+uint8 SIDE_RIGHT=2
+
+uint8 TEAM_UNKNOWN=0
```

Comment:
Are there any leagues where colored goalposts are still a thing? In normal soccer, it is quite hard to decide whether or not a goalpost is from the opponent team or not when we only look at the image space. The happens later of in the world model where more global information is available.

Reply:
I have included this because robocup junior humanoid soccer, a new league in robocup junior uses this. Here's the [RuleBook](https://www.robocupap.org/RCAP2019_Rules/RCAPChallenges/Junior-Humanoid-Soccer.pdf) and a [Video of a Game](https://www.youtube.com/watch?v=AmfKpkL2MUc) from the league.